pkg/ebpf/state.go

```diff
@@ -157,17 +161,33 @@ func (ns *networkState) Connections(id string, latestTime uint64, latestConns []
 }
 
 // getConnsByKey returns a mapping of byte-key -> connection for easier access + manipulation
-func getConnsByKey(conns []ConnectionStats, buf *bytes.Buffer) map[string]*ConnectionStats {
+func getConnsByKey(conns []ConnectionStats, buf *bytes.Buffer) (map[string]*ConnectionStats, int) {
```

Comment:
nit: can you add a comment explaining the int being returned?  It's not obvious from the signature